_plugins/author.rb

```diff
@@ -66,6 +65,13 @@ def with_pic(input)
         "<img class='img-circle team-img bio-clip' src='/assets/images/18f.png' alt='18F logo'>"
       end
     end
+    def lookup(input, args)
```

Comment:
Oh, and should it return nil if the property isn't found, or `''`, or some sort of error placeholder?